### PR TITLE
Foreman 3.0 config updates

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: repoclosure/yum.conf [hunk 1]
```diff
@@ -187,9 +187,9 @@ baseurl=https://copr-be.cloud.fedoraproject.org/results/ndp/qpid/epel-6-x86_64/
 name=Pulp Community Release
 baseurl=https://repos.fedorapeople.org/repos/pulp/pulp/stable/latest/6/x86_64
 
-[el6-foreman-client-nightly]
-name=Foreman Client nightly EL6
-baseurl=http://koji.katello.org/releases/yum/foreman-client-nightly/el6/x86_64/
+[el6-foreman-client-3.0]
+name=Foreman Client 3.0 EL6
+baseurl=http://koji.katello.org/releases/yum/foreman-client-3.0/el6/x86_64/
```
Comment (Odilhao) [inline repoclosure/yum.conf:192]: With the drop of EL6 clients here on this PR for [tool_belt](https://github.com/theforeman/tool_belt/pull/388/), do we still need EL6 clients here?

Comment (pcreech) [reply repoclosure/yum.conf:192]: We don't :D  Removing

Comment (pcreech) [reply repoclosure/yum.conf]: Removed.  Dropped f29 as well for similar reasons

File: rel-eng/releasers.conf [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 
 [koji-foreman-client]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel6 foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.0-rhel6 foreman-client-3.0-rhel7 foreman-client-3.0-el8 foreman-client-3.0-fedora29
 
 [koji-foreman-client-jenkins]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel6 foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.0-rhel6 foreman-client-3.0-rhel7 foreman-client-3.0-el8 foreman-client-3.0-fedora29
```
Comment (Odilhao) [inline rel-eng/releasers.conf:31]: There's also one entry for rhel6 here.

File: rel-eng/releasers.conf [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-plugins-nightly-nonscl-rhel7 foreman-plugins-nightly-rhel7 foreman-plugins-nightly-el8
+autobuild_tags = foreman-plugins-3.0-nonscl-rhel7 foreman-plugins-3.0-rhel7 foreman-plugins-3.0-el8
 builder = tito.builder.FetchBuilder
 builder.rpmbuild_options = --define "foremandist .fm3_0"
 
 [koji-foreman-client]
 releaser = tito.release.KojiReleaser
-autobuild_tags = foreman-client-nightly-rhel6 foreman-client-nightly-rhel7 foreman-client-nightly-el8 foreman-client-nightly-fedora29
+autobuild_tags = foreman-client-3.0-rhel6 foreman-client-3.0-rhel7 foreman-client-3.0-el8 foreman-client-3.0-fedora29
```
[the remark this answers is not in the document]
Comment (pcreech) [reply rel-eng/releasers.conf]: Fixed

File: rel-eng/releasers.conf (hunk 2 above)
Comment (pcreech) [reply rel-eng/releasers.conf]: Fixed

**Review by Odilhao — APPROVED** [review]

**Review by ekohl — APPROVED** [review]
In the past I usually did the Katello part as well during branching.

**Comment by pcreech** [comment]
> In the past I usually did the Katello part as well during branching.

Noted.  Katello hadn't started their process yet so I wasn't sure if it was ok to jump the gun.  Merging, and will add a katello one

**Pull request merged**